napari_allencell_annotator/widgets/annotation_item.py

```diff
@@ -161,29 +174,41 @@ def _type_changed(self, text: str):
         text : str
             the new type selected.
         """
-        default_widget = self.layout.itemAtPosition(0, 7).widget()
-        default_widget.setParent(None)
-        self.layout.removeWidget(default_widget)
+        default_item: QLayoutItem = self.layout.itemAtPosition(0, 7)
+        if default_item is not None:
+            default_widget: QWidget = default_item.widget()
+            default_widget.setParent(None)
+            self.layout.removeWidget(default_widget)
 
         if text == "text":
+            self.default_label.show()
             self.default_options.hide()
             self.default_options_label.hide()
             self.layout.addWidget(self.default_text, 0, 7, 1, 2)
 
         elif text == "number":
+            self.default_label.show()
             self.default_options.hide()
             self.default_options_label.hide()
             self.layout.addWidget(self.default_num, 0, 7, 1, 2)
 
         elif text == "checkbox":
+            self.default_label.show()
             self.default_options.hide()
             self.default_options_label.hide()
             self.layout.addWidget(self.default_check, 0, 7, 1, 2)
-        else:
+
+        elif text == "dropdown":
+            self.default_label.show()
             self.default_options.show()
             self.default_options_label.show()
             self.layout.addWidget(self.default_text, 0, 7, 1, 2)
 
+        else:
```

Comment:
should this be an elif for `point`?

Reply:
fixed in https://github.com/aics-int/napari-allencell-annotator/pull/92/commits/6bfbf5e42452477f6bdbed152dcf5f16c8a90a38